views/templates/errors/404.html.twig

```diff
@@ -0,0 +1,25 @@
+{% extends "@ModuleAutoUpgrade/layouts/error.html.twig" %}
+
+{% block img %}
+  <img class="error-page__img" src="{{ psBaseUri }}img/404.svg" width="408" height="141" alt=""/>
+{% endblock %}
+
+{% block title %}
+  <h2 class="error-page__title">
+    {{ 'Something went wrong...'|trans({}) }}
+  </h2>
+{% endblock %}
+
+{% block description %}
+  <p class="error-page__desc">
+    {{ 'We can’t find your update file in the server.'|trans({}) }}
+    <br/>
+    {{ 'Add a file to restore or update your store.'|trans({}) }}
```

Comment:
This is usually caused by something else. I would rather say something like "The address you requested cannot be found... bla bla"

Reply:
I have updated the wording with the text you proposed on Figma, but I think we may need to make some changes later.